qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?:\/\/.+"/>
```

Comment:
@JimDeanSpivey but it would create more bad cases than good ones, we usually don't add full URLs in the code (why would we?, it's a configuration thing), only in javadocs.